config/routes/api/v2.rb

```diff
@@ -206,6 +206,11 @@
         (resources :organizations, :only => [:index, :show]) if SETTINGS[:organizations_enabled]
         resources :domains, :except => [:new, :edit]
         resources :interfaces, :except => [:new, :edit]
+        resources :parameters, :except => [:new, :edit] do
+          collection do
+            delete '/', :to => :reset
```

Comment:
This syntax is deprecated in Rails, I'm updating it in #3236 for others, but if you could change `:to` to `:action` here it'd be appreciated.